packages/tldraw/src/lib/utils/export/copyAs.ts

```diff
@@ -29,6 +29,10 @@ export function copyAs(
 	if (window.navigator.clipboard.write) {
 		const { blobPromise, mimeType } = exportToBlobPromise(editor, ids, format, opts)
 
+		blobPromise.catch((err) => {
+			console.error(err)
```

Comment:
should this do something more than just write to console? no toast or anything? does it basically silently fail?

Reply:
there's already a toast saying that the copy failed, but without this we get a generic message in the console about it. This shows us the actual cause of the failure in the console